types/node-fetch/index.d.ts

```diff
@@ -12,6 +12,7 @@
 
 import { Agent } from "http";
 import { URLSearchParams } from "url";
+import { AbortSignal } from 'abort-controller';
```

Comment:
Just a nit, seeing that the other imports use double quotes:
```suggestion
import { AbortSignal } from "abort-controller";
```